.changeset/red-frogs-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+fix: Report Custom Build failures as `UserError`s
```

Comment:
I don't believe this is a user-facing change

Reply:
It slightly changes the error message users would see, and doesn't offer them a sentry prompt anymore